src/configuration.rs

```diff
@@ -56,11 +56,16 @@ pub trait VoxelWorldConfig: Resource + Default + Clone {
     /// If you are not using this feature, you can set this to `()`.
     type ChunkUserBundle: Bundle + Clone;
 
-    /// Distance in chunks to spawn chunks around the camera
-    fn spawning_distance(&self) -> u32 {
+    /// Maximum distance in chunks to spawn chunks, whether in view or not, depending on the ChunkSpawnStrategy
+    fn max_spawning_distance(&self) -> u32 {
```

Comment:
I would prefer to keep the existing name for this one, that way we avoid a breaking change here.